Bottom: ced73e2f07ead2345f18c4f0dd6287ae4f931df8
Top:    44961b9c0eba5cc32beade5f3447fa0f40354515
Author: Ben Greear <greearb@candelatech.com>
Date:   2015-02-24 15:56:07 -0800

mac80211-hwsim: remove dmesg spam about get-survey.

This message just fills up dmesg and/or kernel logs and does
not provide any useful information.

Signed-off-by: Ben Greear <greearb@candelatech.com>


---

diff --git a/drivers/net/wireless/mac80211_hwsim.c b/drivers/net/wireless/mac80211_hwsim.c
index 8481105..4789626 100644
--- a/drivers/net/wireless/mac80211_hwsim.c
+++ b/drivers/net/wireless/mac80211_hwsim.c
@@ -1794,7 +1794,7 @@ static int mac80211_hwsim_get_survey(
 {
 	struct ieee80211_conf *conf = &hw->conf;
 
-	wiphy_debug(hw->wiphy, "%s (idx=%d)\n", __func__, idx);
+	/* wiphy_debug(hw->wiphy, "%s (idx=%d)\n", __func__, idx); */
 
 	if (idx != 0)
 		return -ENOENT;
